centreon-gorgone/gorgone/class/clientzmq.pm

```diff
@@ -122,20 +131,49 @@ sub get_connect_identity {
     return $self->{identity} . '-' .  $self->{extra_identity};
 }
 
+sub connect_wait_response {
+    my ($self, %options) = @_;
+
+    my $w1 = $self->{connect_loop}->io(
+        $sockets->{ $self->{identity} }->get_fd(),
+        EV::READ,
+        sub {
+            $self->event(identity => $self->{identity});
+        }
+    );
+
+    # Soemtimes EV library sent timer event directly (it doesn't wait the timeout)
```

Comment:
Est-ce que tu à un script qui reproduit ce problème ?
Je pense que EV n'interragis pas correctement avec zmq, mais je n'ai pour l'instant pas d'éléments concret pour accuser EV plutôt que ZMQ.
------
Do you have a script that reproduce this ?
I think EV don't interract well with zmq, but for now I don't have proof acusing more EV nor ZMQ.